db/migrate/20210512111851_create_saved_pages.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSavedPages < ActiveRecord::Migration[6.1]
+  def change
+    create_table :saved_pages do |t|
+      t.references :oidc_user, null: false
```

Comment:
This is the one, right? There's also a `User` model, but that seemed to be about something else?

Reply:
We discussed this in slack, the `User` table is for sso!